Documentation/kube-aws-approver.md

```diff
@@ -0,0 +1,100 @@
+# Kubernetes AWS approver
+
+A custom node CertificateSigningRequest approver for kubernetes clusters running on AWS.
```

Comment:
Can you link to the `CertificateSigningRequest` definition here

Reply:
done